src/main/proto/tech/figure/servicing/v1beta1/loan_state.proto

```diff
@@ -98,6 +98,8 @@ message LoanState {
   tech.figure.util.v1beta1.Money      principal_paid           = 28; // Sum of amount paid to principal
   tech.figure.util.v1beta1.Money      past_due_amount          = 29; // Amount past due
   tech.figure.util.v1beta1.Money      deferred_principal       = 30; // Deferred Principal balance
+  tech.figure.util.v1beta1.Decimal    credit_utilization_percentage = 31; // Current credit utilization % for HELOC's, for use by investors. (principal balance / credit limit)
+  tech.figure.util.v1beta1.Rate       net_margin_rate          = 32; // Net margin rate with discounts INCLUDED, for use by investors. (total rate - index rate)
```

Comment:
I believe these don't need to be explicit properties since they're only necessary for reporting purposes. They can be added to the KV map instead

Reply:
You're not wrong and I don't have a strong opinion on these two fields

Reply:
Parity is definitely necessary. The difference with loan state is that fields no longer need to be statically defined and instead be put into the kv map. The intent of this new design is to remove the need for these kinds of maintenance PRs and provide flexibility to the publisher's needs to add additional fields as often as they like